src/modelinstance.cpp

```diff
@@ -839,6 +847,7 @@ Status ModelInstance::loadModelImpl(const ModelConfig& config, const DynamicMode
             this->status.setLoading(ModelVersionStatusErrorCode::UNKNOWN);
             return status;
         }
+        loadRTInfo();
```

Comment:
Maybe it would be good idea to merge multiple RTInfo loading logic into one place? We already load rt_info during loadTensors phase where we configure layout. Check ModelInstance::getReportedTensorLayout

Reply:
There we extract rt_info from model input, here directly from ov::Model. For me it is different part of extracting information abourt model (1 is inputs, outputs info, 2 is coming now - rt_info.